nix/deku.nix

```diff
@@ -20,35 +22,32 @@ pkgs.ocamlPackages.buildDunePackage {
 
   inherit doCheck;
 
-  nativeBuildInputs = [ nodejs ] ++ (with pkgs.ocamlPackages; [
-    cmdliner
-    ppx_deriving
-    ppx_deriving_yojson
-    lwt
-    dream
-    mirage-crypto
-    mirage-crypto-pk
-    mirage-crypto-rng
-    mirage-crypto-ec
-    piaf-dream-compat
-    mrmime
-    hex
-    tezos-micheline
-    digestif
-    ppx_blob
-    secp256k1-internal
-    bigstring
-    domainslib
-    utop
-    reason
-  ]);
-
-  propagatedBuildInputs = [ npmPackages ];
-
-  checkInputs = with pkgs.ocamlPackages; [
-    alcotest
-    qcheck
-    qcheck-alcotest
-    rely
-  ];
+  nativeBuildInputs = [ nodejs npmPackages ]
+    ++ (with ocamlPackages; [ utop reason ]);
+
+  propagatedBuildInputs = with ocamlPackages;
+    [
+      cmdliner
+      ppx_deriving
+      ppx_deriving_yojson
+      lwt
+      dream
+      mirage-crypto
+      mirage-crypto-pk
+      mirage-crypto-rng
+      mirage-crypto-ec
+      piaf-dream-compat
+      mrmime
+      hex
+      tezos-micheline
+      digestif
+      ppx_blob
+      secp256k1-internal
+      bigstring
+      domainslib
+      prometheus
+      prometheus-dream
+    ] ++ checkInputs ++ [ npmPackages ];
```

Comment:
Adding checkInputs here is needed if we want to cross-compile or use `strictDeps = true`